docs/3.0rc/resources/upgrade-prefect-3.mdx

```diff
@@ -0,0 +1,226 @@
+---
+title: Upgrade to Prefect 3
+description: Learn how to upgrade from Prefect 2 to Prefect 3.
+---
+
+Some patterns supported by Prefect 2 are no longer supported with Prefect 3, or are enabled differently. 
+This guide covers changes that you may need to make to ensure your Prefect 2 workflows keep flowing.
```

Comment:
I wonder if it might be helpful to callout the the Cloud API isn't changing from Prefect 2-3. Something reassuring about the relatively small migration scope.

Alternative or in addition to the above, we could add a **What stays the same** section like we have for the [worker upgrade guide](https://docs.prefect.io/latest/guides/upgrade-guide-agents-to-workers/#whats-similar). 

I think is helpful for people to get a sense of all the things they've built that won't need to be changed.